sealtk/core/test/ImageUtils.cpp

```diff
@@ -185,17 +188,17 @@ void TestImageUtils::initTestCase()
   m_pixelPackedShader.reset(new QOpenGLShaderProgram);
   QVERIFY(m_pixelPackedShader->create());
   QVERIFY(m_pixelPackedShader->addShaderFromSourceCode(
-    QOpenGLShader::Vertex, VERTEX_SHADER_CODE));
+            QOpenGLShader::Vertex, VERTEX_SHADER_CODE));
```

Comment:
I've always preferred to indent continuations two spaces after the beginning of the previous line, regardless of where the actual method call starts (this is how it's done in CMake with clang-format.) Is there a way to tell Uncrustify about this?

Reply:
Assuming there are no arguments on the previous line of course.

Reply:
It looks like 'no'. The `indent_continue` and `indent_func_call_param` options are relevant, but it appears uncrustify cannot apply them iff there are no params on the previous line (which we definitely want). (Which is unfortunate, because I'm inclined to agree with your preference, influenced no doubt in that that's how katepart wants to indent...)

Feature request? :slightly_smiling_face:

Reply:
Considering that we're still a long way off from a fully Uncrustified SEAL-TK, and considering that this doesn't bring us much closer, I'm inclined to ignore Uncrustify's recommendation and keep this (and other similar instances) unchanged for now.

Reply:
Actually, the more I think about it, the more I think I *do* want to keep this style.

It's not as bad for nested function parameters, but in some cases, the "single indent" style can look pretty odd. Compare [old](https://github.com/Kitware/seal-tk/blob/d9d4d62d9c404786ea45c7a042407a3b9809d4c6/sealtk/core/test/KwiverPipelineWorker.cpp#L249-L255) vs. [new](https://github.com/Kitware/seal-tk/blob/aade9a88aceb18fcef20c8e07d1899a91c8c75aa/sealtk/core/test/KwiverPipelineWorker.cpp#L247-L253).